std/algorithm/sorting.d

```diff
@@ -2994,6 +2994,165 @@ unittest
     }
 }
 
+// medianOf
+/**
+Computes the median of 2 to 5 arbitrary indexes in random-access range `r`
```

Comment:
Nitpick: The individual sections of the docs would look clearer (in the source code) if they were indented like the docs of [`topNIndex`](https://github.com/dlang/phobos/blob/v2.071.2/std/algorithm/sorting.d#L2511) above or [`nextPermutation`](https://github.com/dlang/phobos/blob/v2.071.2/std/algorithm/sorting.d#L2640) below.